### PR TITLE
add about tab to settings modal

### PR DESCRIPTION
**Pull request by bbashy**
## Description
Added an about page to the settings modal with some info taken from the README (now in an ABOUT.md).

## Related Issue
#654 

## How Has This Been Tested?
I've developed this with a fully running instance.

## Screenshots (if appropriate):
Will update with one.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by bbashy** [comment]
Something has broken since I last did this (rebased recent changes) and I can't work out what

```
Error: TextRenderer(...): A valid React element (or null) must be returned. You may have returned undefined, an array or some other invalid object
```

File: client/src/javascript/components/modals/settings-modal/AboutTab.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+import SettingsTab from './SettingsTab';
+
+export default class AboutTab extends SettingsTab {
+  state = {
+    about: null,
+  };
+
+  componentWillMount() {
+    fetch(AboutMarkdownPath)
```
Comment (jfurrow) [inline client/src/javascript/components/modals/settings-modal/AboutTab.js]: 1. Is this going to work for users who have the `baseURI` config set in `config.template.js`? I can't remember if Webpack will handle this automatically not.
2. `fetch` is lovely but we should use `axios` for consistency — [here](https://github.com/jfurrow/flood/blob/master/client/src/javascript/actions/SettingsActions.js#L51-L53) is an example of a GET request.

Comment (bbashy) [reply client/src/javascript/components/modals/settings-modal/AboutTab.js]: 1. This path should always be correct unless the path of `client/src/javascript/components/modals/settings-modal/AboutTab.js` moves.
2. Can't use axois for non-HTTP requests?

Comment (jfurrow) [reply client/src/javascript/components/modals/settings-modal/AboutTab.js]: Webpack is treating this as a static external resource, it doesn't become part of the JS bundle. `fetch` is fetching the static resource and parsing it, so this should use `axios`.

Comment (bbashy) [reply client/src/javascript/components/modals/settings-modal/AboutTab.js]: Makes sense now. It's actually calling it from a file.

I'm not sure if `baseURI` will affect this.

**Comment by jfurrow** [comment]
@bbashy Changes look good but failing CI build

**Comment by noraj** [comment]
Node 8 and 10 CI image seems broken:
image
image

Node 9 fails at build:
image

**Comment by bbashy** [comment]
Can you retry them for this PR? They did this a few times - it's Travis, not the code.

**Comment by jeanralphaviles** [comment]
Looks like this PR caused #784 due to the ABOUT.md reference in [AboutTab.js](https://github.com/Flood-UI/flood/blob/5065ab2f89ca3d306aaf3a299eedd3853859f1c0/client/src/javascript/components/modals/settings-modal/AboutTab.js#L5). I suspect that ABOUT.md needs to be copied over in the Dockerfile.

**Pull request merged**